databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -37,6 +38,16 @@ public class DatabricksConfig {
       sensitive = true)
   private String clientSecret;
 
+  @ConfigAttribute(value = "scopes", env = "DATABRICKS_SCOPES", auth = "oauth", sensitive = true)
+  private List<String> scopes;
+
+  @ConfigAttribute(
+      value = "redirect_url",
+      env = "DATABRICKS_REDIRECT_URL",
+      auth = "oauth",
+      sensitive = true)
```

Comment:
Ditto, probably not sensitive.

Reply:
The validation logic is case sensitive: https://src.dev.databricks.com/databricks/universe/-/blob/common/authentication/oauth/OAuthAuthorizationEndpointHandler.scala?L91:41

Reply:
Updated